model/scheduler_stats.go

```diff
@@ -139,6 +140,14 @@ func hasTaskId(taskId string, dependsOn []task.Dependency) bool {
 		if d.TaskId == taskId {
 			return true
 		}
+		if d.TaskId == "" {
+			grip.Debug(message.Fields{
```

Comment:
This log message wouldn't really be helpful since it doesn't tell us the task that has the bad state.